usaspending_api/common/mixins.py

```diff
@@ -254,6 +256,70 @@ class SuperLoggingMixin(LoggingMixin):
 
     events_logger = logging.getLogger("events")
 
+    """Mixin to log requests - customized to disable DB logging, remove this method to re-enable"""
+    def initial(self, request, *args, **kwargs):
+        """Set current time on request"""
+
+        # check if request method is being logged
+        if self.logging_methods != '__all__' and request.method not in self.logging_methods:
+            super(LoggingMixin, self).initial(request, *args, **kwargs)
+            return None
+
+        # get IP
```

Comment:
Sure, but my question is more about--do we, in our application logs, want to be the keepers of said IP addresses? Unless we need them for debugging, I'd prefer to take them out. Sorry for not catching this earlier!